apps/fioapp/hack_cmd.h

```diff
@@ -0,0 +1,48 @@
+#ifndef AKLITE_APPS_FIOAPP_HACKCMD_H
+#define AKLITE_APPS_FIOAPP_HACKCMD_H
+
+#include "cmd.h"
+
+#include <boost/algorithm/string/classification.hpp>
+#include <boost/algorithm/string/split.hpp>
+#include "logging/logging.h"
+
+namespace apps {
+namespace fioapp {
+
+extern int hackDockerStore(const std::vector<std::string>& shortlist, const std::string& store_root,
```

Comment:
should we put this in a header to avoid `extern` stuff that might get brittle?

Reply:
I just didn't want to add a whole new file for sake of a single function :).